src/main/java/com/google/sps/servlets/AvailabilityServlet.java

```diff
@@ -31,7 +56,27 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        response.setContentType("plain/text");
-        response.getWriter().println("To be implemented");
+        // Get the id of the tutor whose availability will be displayed.
+        String tutorID = Optional.ofNullable(request.getParameter("tutorID")).orElse(null);
+
+        List<TimeRange> timeslots = new ArrayList<TimeRange>();
+
+        for (Tutor tutor : hardcoded) {
+            if (tutorID.equals(tutor.getEmail())) {
```

Comment:
```suggestion
            if (tutorID.toLowerCase().equals(tutor.getEmail().toLowerCase())) {
```
Make it case insensitive?

Reply:
I agree. I'll submit it as new commit here.